FIAT/expansions.py

```diff
@@ -390,7 +424,11 @@ def get_expansion_set(ref_el):
 def polynomial_dimension(ref_el, degree):
     """Returns the dimension of the space of polynomials of degree no
     greater than degree on the reference element."""
-    if ref_el.get_shape() == reference_element.LINE:
+    if ref_el.get_shape() == reference_element.POINT:
+        if degree > 0:
+            raise Exception("Only degree zero polynomials supported on point elements.")
```

Comment:
Be more specific on exception type, for example: `ValueError`

Reply:
Sure, I can, but I'm just following the convention in the rest of `FIAT/expansions.py` by raising `Exception`. What's preferable - to follow the convention in the file or update to be more specific in this case?

Reply:
Much of FIAT is quite old code, and could be improved in ways that are not necessarily critical to functionality.  It is better to cleanup old code that you have to touch anyway than to follow suboptimal (and perhaps inconsistent among different parts of the codebase) habits.

The argument against cleaning up here is that that would clutter that addition of new functionality in this PR, making it somewhat harder to review.  The best would be to file a separate pull request which cleans up the exception types in `FIAT/expansions.py`.

Reply:
Thanks. I've updated the where I've touched code in 33bfcbf48d8fa2f28cb3c0d080417fce2635f7ce . I'll consider a PR in due course.